<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fight</class>
 <widget class="QDialog" name="fight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="playerLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>231</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="enemyLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>231</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="attack3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Attack</string>
   </property>
  </widget>
  <widget class="QPushButton" name="attack2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Attack</string>
   </property>
  </widget>
  <widget class="QPushButton" name="attack1">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Attack</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="playerHp">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="enemyHp">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Health:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="healButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Heal</string>
   </property>
  </widget>
  <widget class="QLabel" name="actionsDoneLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
